analysis/grammar_search_analysis.py

```diff
@@ -61,6 +89,7 @@ def _run_proxy_analysis(env_names: List[str], score_function_names: List[str],
             GripperOpen, OnTable, HoldingTop, HoldingSide, Holding, IsWet,
             IsDry, IsDirty, IsClean
         }
+        # Example of how to add an extra predicate.
```

Comment:
wait, what exactly is the example here? these all look like built in predicates to me

Reply:
is this comment supposed to be moved up?